packages/mwp-app-route-plugin/src/handler.js

```diff
@@ -15,28 +15,29 @@ import url from 'url';
  *    the Vary header to 'User-Agent', in order for the google bots
  *    to crawl mobile and desktop versions of the site
  */
-export default (languageRenderers: { [string]: LanguageRenderer }) => (
+export default (languageRenderers: { [string]: LanguageRenderer }): any => (
```

Comment:
@mmcgahan not sure how to flow type the response here, on line 21 for the curried method, and line 30 for the handler function. Thoughts?

Reply:
the outer function returns a function

```js
export default (languageRenderers: { [string]: LanguageRenderer }): (HapiRequest, HapiResponseToolkit) => Promise<HapiResponseToolkit | Error> => (
```

and the inner function is the same thing, although I don't think you need to declare the return value - just let Flow infer it from the outer function's type